src/assets/scss/application.scss

```diff
@@ -2,3 +2,9 @@ $govuk-assets-path: "/public/";
 
 @import "../../../node_modules/govuk-frontend/govuk/all";
 @import "../../../node_modules/hmpo-components/all";
+@import "../../../node_modules/@govuk-one-login/one-login-language-toggle/stylesheet/styles";
+
+// style for language toggle
+.last-item {
```

Comment:
This can be removed as the latest versionof the toggle handles the styling

Reply:
ahh ok I have removed now